test-poc/framework/src/main/java/org/keycloak/test/framework/webdriver/ChromeWebDriverSupplier.java

```diff
@@ -0,0 +1,42 @@
+package org.keycloak.test.framework.webdriver;
+
+import org.keycloak.test.framework.injection.InstanceWrapper;
+import org.keycloak.test.framework.injection.LifeCycle;
+import org.keycloak.test.framework.injection.Registry;
+import org.keycloak.test.framework.injection.Supplier;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class ChromeWebDriverSupplier implements Supplier<WebDriver, TestWebDriver> {
+
+    @Override
+    public Class<TestWebDriver> getAnnotationClass() {
+        return TestWebDriver.class;
+    }
+
+    @Override
+    public Class<WebDriver> getValueType() {
+        return WebDriver.class;
+    }
+
+    @Override
+    public InstanceWrapper<WebDriver, TestWebDriver> getValue(Registry registry, TestWebDriver annotation) {
+        ChromeDriver driver = new ChromeDriver();
```

Comment:
Minor nit, we can use new fancy Java features such as type-inference here.
```suggestion
        final var driver = new ChromeDriver();
```

Reply:
I don't think it is very useful in this case as it only adds extra work for the compiler and doesn't improve readability that much

Reply:
Since we have no official code style it doesn't matter, this is all pure personal preference, feel free to ignore it. The cost for the compiler here is minimal though since the right side operation has a clear type, so it's not like it has to do a wide inference.